api/v1alpha1/recipe_types.go

```diff
@@ -22,14 +22,19 @@ type RecipeSpec struct {
 	//+listType=map
 	//+listMapKey=name
 	Groups []*Group `json:"groups"`
+	// Volumes to protect from disaster
+	//+optional
+	Volumes *Group `json:"volumes"`
```

Comment:
If I understand the intention here, it's to use a single group for Volumes and separate them out from Groups, eventually switching the type of Groups to be split into an actual "Group" type, and a "Volume" type, which currently exists as a union of both structs. I'm fine with this, but is there a chance we'd ever need more than one Volume group? Or could we simply define all labels required for all applications in the target Namespace in a single Volume group?

Reply:
Yes, the intent is to have a single group for volumes as that matches the single `PVCSelector` field in the VRG Spec.  Another recipe could be defined with a different volume group, i.e., PVC label selector, if it varies by workload.